scripts/presubmit.sh

```diff
@@ -12,7 +12,7 @@ find . -name "mock_*" -delete
 golint --set_exit_status ./...
 go vet ./...
 misspell -error -locale US .
-nolicense=$(find . -name "*.go" | xargs grep -L "Apache License")
+nolicense=$(find . -path ./protoc -prune -name "*.go" -o -name "*.proto" | xargs grep -L "Apache License")
```

Comment:
I think maybe stick to your first suggestion, to avoid everyone having to look up obscure `find` options...